apps/ops/const.py

```diff
@@ -85,3 +85,8 @@ class JobStatus(models.TextChoices):
 class Scope(models.TextChoices):
     public = 'public', pgettext_lazy("scope", 'Public')
     private = 'private', _('Private')
+
+
+class FieldType(models.TextChoices):
+    text = 'text', _('Text')
+    select = 'select', _('Select')
```

Comment:
No known issues in the provided code snippet that would require additional comments or explanations. The use of choices for various fields is consistent with standard Python usage, and there appear to be no errors or oddities at this point.

The most important points to note:

1. Use case (JobStatus.choices) follows convention well.
 
2. No specific optimizations or improvements have been suggested here; however, using TextChoices ensures readability by providing clear names.

3. Class `FieldType` appears unnecessary as a subclass of `models.TextChoices`, possibly implying an oversight in the structure of the project documentation. If it's meant as clarification for developers, it makes sense to use it correctly when needed elsewhere within the codebase.

4. A potential suggestion might involve creating separate utility functions or classes around these models if they will be used frequently, like converting between different formats or handling cases where both private/public statuses apply.

Remember I cannot execute code directly and do not see actual instances of the models being utilized or how often/where they are accessed. This analysis focuses on general patterns and design guidelines rather than immediate implementation issues. For instance, missing imports, redundant docstrings, or non-compliance with PEP standards should also be considered.